docs/book/src/developer/tilt.md

```diff
@@ -192,6 +192,28 @@ COPY --from=tilt-helper /usr/bin/docker /usr/bin/docker
 COPY --from=tilt-helper /go/kubernetes/client/bin/kubectl /usr/bin/kubectl
 ```
 
+
+**extra_args** (Object, default={}): A mapping of provider to additional arguments to pass to the main binary configured
+for this provider. Each item in the array will be passed in to the manager. 
+
+Example:
+
+```json
+{
+    "extra_args": {
+        "core": ["--feature-gates=MachinePool=true"],
+        "kubeadm-bootstrap": ["--feature-gates=MachinePool=true"],
```

Comment:
does this actually do anything? I'm not using it AFAIK, but I don't recall if kubeadm-bootstrap specifically uses this value at all

Reply:
It looks like it's used for adding the watch on MachinePool: https://github.com/kubernetes-sigs/cluster-api/blob/99a95aaa59492924b04e6dddbea17992fecca24d/bootstrap/kubeadm/controllers/kubeadmconfig_controller.go#L109-L116 and also in determining the config owner (which is used during the Reconcile method): https://github.com/kubernetes-sigs/cluster-api/blob/99a95aaa59492924b04e6dddbea17992fecca24d/bootstrap/util/configowner.go#L95-L100

Reply:
thanks for the links, probably should have tried a simple ctrl-f 😋